core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.ts

```diff
@@ -20,6 +20,8 @@ require('filters/FormatTimerFilter.ts');
 require('pages/exploration_editor/ExplorationStatesService.ts');
 require(
   'pages/exploration_editor/translation_tab/TranslationLanguageService.ts');
+require(
+  'pages/exploration_editor/translation_tab/VoiceoverRecordingService.ts');
```

Comment:
Please try to keep the requires sorted. So move this `require()` after the one below.

Reply:
okay will sort them alphabetically.

Reply:
Should be done.